code/modules/reagents/chemistry/machinery/chem_dispenser.dm

```diff
@@ -53,36 +53,39 @@
 		/datum/reagent/oxygen,
 		/datum/reagent/phosphorus,
 		/datum/reagent/potassium,
-		/datum/reagent/uranium/radium,
 		/datum/reagent/silicon,
 		/datum/reagent/silver,
 		/datum/reagent/sodium,
+		/datum/reagent/iron,
+		/datum/reagent/water,
+		/datum/reagent/oil,
+		/datum/reagent/uranium/radium,
 		/datum/reagent/stable_plasma,
-		/datum/reagent/consumable/sugar,
 		/datum/reagent/sulfur,
```

Comment:
I'm not sure if these ,s are being added to the end of each list for a reason or not

```suggestion
		/datum/reagent/sulfur
```

Reply:
leftover from when I was trying to find what was causing the box bar spawner to shit itself (it turned out it was chemical reactions